data/json/vehicleparts/frames.json

```diff
@@ -83,19 +83,18 @@
     "durability": 100,
     "requirements": {
       "install": {
-        "skills": [ [ "mechanics", 0 ] ],
+        "skills": [ [ "survival", 0 ] ],
         "time": 60000,
         "components": [ [ [ "rope_6", 1 ] ] ]
       },
       "removal": {
-        "skills": [ [ "mechanics", 1 ] ]
+        "skills": [ [ "survival", 0 ] ]
```

Comment:
Ah right. Do I omit the number if having it train survival skill is desired behavior?